BrowserCommunication.js

```diff
@@ -131,6 +131,25 @@ export function addListener(messageType, callback) {
     callbacks[messageType].push(callback);
 }
 
+/**
+ * Remove an existent listener of a specific type.
+ *
+ * @public
+ * @param {COMMUNICATION_MESSAGE_TYPE} messageType type of message to receive
+ * @param {listenerCallback} callback
+ * @returns {void}
+ */
+export function removeListener(messageType, callback) {
+    checkMessageTypeVadility(messageType);
+
+    if (!(messageType in callbacks)) {
+        return;
+    }
+
+    callbacks[messageType] = callbacks[messageType]
```

Comment:
This re-creates the whole array (and re-iterates the whole one), but yeah, I guess this is okay. It may not be performance-optimized, but it's easy to read, so that's also good. :smile:

Reply:
Yeah, I could change it to slice (or splice) and alter the array if you want. Usually, I don't use those on my daily basis so the filter was the one to go for me. My mindset is directed at avoiding side-effects haha

Reply:
ok